### ISSUE TITLE
Confused about GetServiceTicket

### ISSUE THREAD

**Issue by Vaccano** [SteveSyfuhs/Kerberos.NET#214]
I am trying to get a base 64 Kerberos Ticket for my test user so I can automate some tests.

The samples show to call `client.GetServiceTicket`, but that call expects an SPN value.  I looked up what an SPN is and I don't think that my test user has any of them.  (A call to `setspn -L "myDomain\myTestUser"` returned an empty list.)

But when do a _Run as different user_ on Chrome and let the browser do integrated windows authentication, it works.  (So it seems it can get a Kerberos Token somehow.)

**I may be trying to do something that this library does not help with, but I thought I would ask, is there way to get a Kerberos token that can be passed in a header (like `Authorize: Negotiate KESWFS.......`) WITHOUT an SPN on the user?**

**Comment by SteveSyfuhs** [comment]
User's don't have SPNs. SPNs are _service_ principal names. The SPN is for the thing the user is getting a ticket to. So you're connecting to `https://someserver.com/`. The browser or HTTP client is going to infer what the SPN is based on the host name and service, and pick `http/someserver.com`. The client will send a request to Active Directory and ask for a service ticket for that particular SPN. In AD that SPN is registered to the service account or machine for that particular endpoint.

Active Directory then generates a service ticket and encrypts it to the password of the service or machine account that has the SPN registered. AD returns it to the client, the client encodes it into the header, the service receives it, and the service decrypts it using it's service or machine password that was configured separately.

In other words from the client's perspective you're not asking for a generic ticket to nothing/everything (that's what `.Authenticate(Credential...)` does). GetServiceTicket needs to know what service you want a ticket to.

**Comment by Vaccano** [comment]
WOW!  Thank you for the incredibly fast response!

That cleared it up for me.  I put in:

    var ticket = await client.GetServiceTicket("http/myServerThatWantsTheKerberosToken.myDomain.net");

And got a ticket back!

Thank you so much for the fantastic library and for the amazingly fast response!
